vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh

```diff
@@ -34,7 +34,7 @@ fi
 sig_resource_id="/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP_NAME}/providers/Microsoft.Compute/galleries/${SIG_GALLERY_NAME}/images/${SIG_IMAGE_NAME}/versions/${SIG_IMAGE_VERSION}"
 disk_resource_id="/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP_NAME}/providers/Microsoft.Compute/disks/${CAPTURED_SIG_VERSION}"
 
-if [[ ${OS_TYPE} == "Linux" && ${ENABLE_TRUSTED_LAUNCH} == "True" ]]; then
+if [[ ("$OS_TYPE" == "Linux" || "$OS_SKU" == "CBLMariner") && "$ENABLE_TRUSTED_LAUNCH" == "True" ]]; then
```

Comment:
the extra condition on CBLMariner adds nothing?

Reply:
duh uh, yes. for some reason, I thought it was Ubuntu and not Linux. thanks!